lo-la-do-li/devWit
# Testing

### What’s this PR do?
- Adds unit and integration tests to all components within DevWit App
- Adds propTypes to JokesContainer and JokeCard 
### Where should the reviewer start?
- All test files:  lines following imports
- JokeCard: 44
- JokesContainer: 26
### How should this be manually tested?
- `npm test` select 'a' to run all tests, or 'p' to search a test by pattern. They should all be passing.
### Any background context you want to provide?
- There are minor changes to CSS files, but this does not need to be reviewed 
### Screenshots (if appropriate):
n/a
### Questions:
Does the testing cover all component rendering and async functions?
